splinterd/Cargo.toml

```diff
@@ -41,6 +41,7 @@ cylinder = "0.2"
 flexi_logger = "0.14"
 health = { path = "../services/health", optional = true }
 log = "0.4"
+log4rs = {version ="1.0", features = ["pattern_encoder","delete_roller","compound_policy","size_trigger","file_appender","toml_format","config_parsing"]}
```

Comment:
This should be optional for now and pulled in with the experimental feature.

Note the spacing is incorrect -- "{version" vs. "{ version" and speaces after commas, etc. to match the style used elsewhere.

Version should be set to "1" not "1.0" as they are functionally equivalent and we prefer the shorter.

Are all of these features needed explicitly? This is very strange.

Reply:
fixed